### PR TITLE
Add installation instruction.

### PR DESCRIPTION
**Pull request by yuliusardian**
Hi @dvesh3 @fashxp, I added some installation instruction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,24 @@ follow the instructions:
...
 
+# Installation
+
+On your Pimcore X root project
+
+```bash
+$ composer -vvv require pimcore/bundle-generator
+```
+
+After that at `src/Kernel.php` on `registerBundlesToCollection` function register the bundle by using the following code snippet :
```
Comment (fashxp) [inline README.md]: the preferred way would be to go the standard pimcore way and use Pimcore extension manager or  `bin/console pimcore:bundle:enable` command for enabling the bundle... so there is no need to change the `Kernel` class...

**Review by fashxp — CHANGES_REQUESTED** [review]
thank you very much for your PR ... could you please sign the CLA by adding the comment `I have read the CLA Document and I hereby sign the CLA` to the conversation and update the instructions according to my comment...

File: README.md (hunk 1 above)
Comment (yuliusardian) [reply README.md]: I have read the CLA Document and I hereby sign the CLA

Comment (yuliusardian) [reply README.md:19]: Yes,  Previously I did enable using command, but it is not working, and then I try to register it manually on Kernel class.

**Comment by yuliusardian** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by yuliusardian** [comment]
@fashxp Thanks for the correction, Already update installation instruction into standard Pimcore way.

Happy to contribute, I'm hope I can contribute more. 

Love Pimcore so much. <3

**Review by fashxp — APPROVED** [review]

**Comment by fashxp** [comment]
hmm, it seems that the email used for the PR is not linked to your github account ... 
see also https://patch-diff.githubusercontent.com/raw/pimcore/bundle-generator/pull/3.patch and instructions in the github-actions comment above. 

thx again.

File: README.md [hunk 2]
```diff
@@ -8,6 +8,22 @@ follow the instructions:
 - `bundle:generate` - to generate default symfony bundles
 - `pimcore:generate:bundle` - to generate Pimcore bundles
 
+# Installation
+
+On your Pimcore X root project
+
+```bash
+$ composer -vvv require pimcore/bundle-generator
```
Comment (jdreesen) [inline README.md]: Why `-vvv`?

```suggestion
$ composer require pimcore/bundle-generator
```

Comment (yuliusardian) [reply README.md]: Pardon my habits,

**Comment by fashxp** [comment]
thx very much

**Pull request merged**